<?xml version="1.0"?>
<launch>

    <!-- external or mavros -->
    <node pkg="bluerov2common" type="EKFDVL" name="EKFDVL" output="screen" required="false">

    </node>


    <node name="dynamic_reconfigure_load" pkg="dynamic_reconfigure" type="dynparam" args="load /EKFDVL $(find bluerov2common)/config/ekfRobot.yaml" />
</launch>
